browser/css/jsdialogs.css

```diff
@@ -568,6 +573,25 @@ td.jsdialog > [id^='table-box']:not(.sidebar) {
 	/* grid-column: '1 / ' + (this._columns + 1) - set inside JS */
 }
 
+#ServerAuditDialog .ui-treeview-headers {
```

Comment:
I have noticed that somehow the .ui-treeview-headers are being set from JS side with a grid-column: 1 / 5; which means that headers are being set in terms of grid-column-end with one extra column (5) instead of (4) as we have in the .ui-treeview-entry.

So, we either try to understand why that is happening on the JS side or for we fix it for this dialog byt adding the following via css into this `#ServerAuditDialog .ui-treeview-headers`:

```css
grid-column-end: 4 !important;
```